code/game/objects/items/weapons/tools.dm

```diff
@@ -151,10 +152,9 @@
 	create_reagents(max_fuel)
 	reagents.add_reagent("fuel", max_fuel)
 	update_icon()
-	return
 
 /obj/item/weapon/weldingtool/examine(mob/user)
-	if(..(user, 0))
+	if(..(user, 1))

```

Comment:
Does it? I thought that setting the second parameter to 1 means that this examination text only shows up if the user is on the same tile. If it's set to 0 I thought it always showed up.

Let me double check this.